internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -21,6 +21,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
```

Comment:
Nit: This should be down with the other runtime imports.

Reply:
Also if we're importing the entire `reconciler/managed` package just for the connection propagation stuff (here and in the composite reconciler too I think) that may be a sign it should move to `crossplane-runtime/pkg/resource`. Not a blocker for this PR though.